tests/wpt-harness/pre-harness.js

```diff
@@ -14,6 +14,7 @@ addEventListener = function addEventListener_wpt(type, handler) {
   }
 }
 
+globalThis.crypto.subtle.generateKey = function () {return Promise.reject(new Error('globalThis.crypto.subtle.generateKey unimplemented'))}
```

Comment:
We need to have this stub implementation of `generateKey` because otherwise the web-platform-tests for hmac functionality will throw an uncaught execption